rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "aws-smithy-http-server"
+version = "0.1.0"
+authors = ["Matteo Bigoi <matbigoi@amazon.com>", "David Pérez <davidpz@amazon.com>"]
```

Comment:
you should probably make `smithy-rs-server` or similar email alias and include it in the author list of the crates (we do this with `aws-sdk-rust@amazon.com`

Reply:
Will do, thanks.